tsdb/head.go

```diff
@@ -1902,7 +1902,10 @@ type memSeries struct {
 
 	mmMaxTime int64 // Max time of any mmapped chunk, only used during WAL replay.
 
-	nextAt int64 // Timestamp at which to cut the next chunk.
+	nextAt                                 int64 // Timestamp at which to cut the next chunk.
+	nativeHistogramChunkBytesOverhead      int   // Captures the number of bytes it takes to store the header and first sample of a native histograms chunk.
+	nativeHistogramChunkFirstSampleTS      int64 // Captures the timestamp of the first sample of a native histograms chunk.
+	nativeHistogramChunkHasComputedEndTime bool  // True if nextAt has been predicted for the current native histograms chunk; false otherwise.
```

Comment:
This feels like a lot to track for just a bit of return. How about just not counting the bytes of the first sample and then count the bytes of the second sample twice?

Reply:
I like it--will do!

Reply:
Hm, actually I had a tough time isolating the size of individual samples that aren't the first sample in the chunk so I just took it all out completely. I think this will result in chunk sizes that are smaller than the target on average but maybe that's ok?

I'll ruminate on this some more though.